src/patternfly/layouts/Page/page.scss

```
@@ -169,28 +120,28 @@
 
 // Link
 .pf-l-page__header-brand-link {
-  display: none;
+  display: flex;
   flex: 1;
   align-items: center;
-  align-self: flex-end;
-  min-height: var(--pf-l-page__header-brand-link--MinHeight); // Hardcode link min-height so it matches the button height
 
-  @media (min-width: $pf-global--breakpoint--sm) {
-    display: flex;
+  .pf-c-brand {
+    max-height: var(--pf-l-page__header-brand-link-c-brand--MaxHeight); // Hardcode brand image max-height so it always aligns with everthing else.
```

What do you think of having 2 `-`'s before a component (2 characters before any new css block)? We seem to follow that structure. `__` for a new element, `--` for a new mq, state, pseudo element.

```suggestion
    max-height: var(--pf-l-page__header-brand-link--c-brand--MaxHeight); // Hardcode brand image max-height so it always aligns with everthing else.
```